llpc/lower/llpcSpirvLowerRayTracingBuiltIn.cpp

```diff
@@ -165,7 +165,13 @@ bool SpirvLowerRayTracingBuiltIn::runImpl(Module &module) {
         Type *instTy = inst->getType();
         if (isa<PointerType>(instTy) && instTy->getPointerAddressSpace() == SPIRAS_Input) {
           assert(isa<GetElementPtrInst>(inst) || isa<BitCastInst>(inst));
-          Type *newInstTy = PointerType::get(instTy->getContainedType(0), SPIRAS_Private);
+          // TODO: Remove this when LLPC will switch fully to opaque pointers.
+          // After transition 'else' will be removed.
+          Type *newInstTy = nullptr;
+          if (instTy->isOpaquePointerTy())
+            newInstTy = PointerType::get(m_builder->getContext(), SPIRAS_Private);
+          else
+            newInstTy = PointerType::get(instTy->getContainedType(0), SPIRAS_Private);
```

Comment:
I think this is equivalent to `PointerType::getWithSamePointeeType(instTy, SPIRAS_Private)`.

Reply:
Right, thanks !